### PR TITLE
add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by MoRaad97**
In this PR we:

Add "Contact Form" section Html and CSS code.
Add the necessary images.
Add the "Footer" section.
Used unordered list tags to build the structure of the "footer" Section content.
Add the "footer" Section style.

### CODE REVIEW

File: css/master.css [hunk 1]
```diff
@@ -334,3 +334,93 @@
...
+
+.contact .contact-header {
+  font-size: 32px;
+  font-weight: bold;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 60px;
+}
+
+.contact form .text-email input {
```
Comment (BenMukebo) [inline css/master.css]: - [x] It's one of the requirements to use a CSS Flexbox property to layout the form elements, please add it.

File: css/master.css [hunk 2]
```diff
@@ -334,3 +334,93 @@
...
+  font-family: "Inter", sans-serif;
+}
+
+.contact form .submit {
+  display: flex;
+  justify-content: center;
+}
+
+.contact form .submit input {
+  padding: 13px;
```
Comment (BenMukebo) [inline css/master.css:395]: - [x] The Button interactions such as hover, active (when it's pressed), are styled as shown in the Figma template. Please make this button style effect  ⬇️ 

buttonfine

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️  
- The form is submitted successfully with valid input values. ✔️  
- Description Pull Request 👍🏻 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Kindly, this is a group project. Let's consider including all the authors in the README file.
- [x] Please let's make sure that contribution to a group project is distributed equally among the members( commit messages should be fairly equal).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/master.css [hunk 3]
```diff
@@ -334,3 +334,112 @@
   padding-top: 30%;
   text-align: center;
 }
+
+.contact {
+  margin-top: var(--section-main-padding);
+  margin-bottom: var(--section-main-padding);
+}
+
+/* stylelint-disable */
```
Comment (danmainah) [inline css/master.css:343]: - [ ] [OPTIONAL] Kindly make sure not to disable linters, its not good practise to disable them unless specified in the project requirements.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @MoRaad97 and @Mugiira-Kim , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️

- [x] Kindly make sure to include all the authors' names in the `readme` file as requested by the previous reviewer.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @MoRaad97 , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**